doc/bevindingen-fase-3.md

```diff
@@ -45,18 +45,52 @@ Zie ook [Topsector-Logistiek/Apache-Pulsar-Auth-Plugin](https://github.com/Topse
 
 ## Delegation-Evidence bij uitbesteding aan derde
 
-Toegang tot het topic voor een transporteur waaraan uitbesteed is, is net zo omslachtig als in fase 2, hiervoor worden extra policies en HTTP headers nodig om Pulsar duidelijk te maken namens wie zij toegang tot het topic wensen te krijgen.
+Toegang tot het topic voor een transporteur waaraan uitbesteed is, is net zo omslachtig als in fase 2. Hiervoor zijn extra policies en HTTP headers nodig om Pulsar duidelijk te maken namens wie zij toegang tot het topic wensen te krijgen.
+
+Dit is in deze demo niet geïmplementeerd omdat alleen een directe transportopdracht al genoeg bevindingen heeft opgeleverd om bij stil te staan.  De extra stap tot uitbesteding voegt daar niets aan toe.
 
 ## Data bij de bron
 
-Om te zorgen dat events zo min mogelijk data bevatten wordt alleen een URL gepubliceerd.  Dat betekent dat de partij die een event publiceert ook een endpoint beschikbaar moet stellen om deze te downloaden. Daarnaast moet een iSHARE token-endpoint aangeboden worden ter authenticatie voor de download.
+Om te zorgen dat events zo min mogelijk data bevatten, wordt alleen een URL gepubliceerd.  Dat betekent dat de partij die een event publiceert ook een endpoint beschikbaar moet stellen om deze te downloaden. Daarnaast moet een iSHARE token-endpoint aangeboden worden ter authenticatie voor de download, aangezien iSHARE authenticatie een mechanisme is dat al bij alle componenten bekend is.
```

Comment:
Wees duidelijker over wat een "event" is en wat een "notification". Het is ook geen doel om events zo weinig mogelijk data te laten bevatten.